README.md

```diff
@@ -71,11 +71,11 @@ Please use the [issue tracker](https://github.com/flosch/pongo2/issues) if you'r
 
 **Deprecated versions** (not supported anymore): v1
 
-| Topic                                | Status                                                                                 |
-| ------------------------------------ | -------------------------------------------------------------------------------------- |       
-| Django version compatibility:        | [1.7](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/)                  |
-| *Missing* (planned) **filters**:     | none ([hints](https://github.com/flosch/pongo2/blob/master/filters_builtin.go#L3))     | 
-| *Missing* (planned) **tags**:        | none ([hints](https://github.com/flosch/pongo2/blob/master/tags.go#L3))                |
+| Topic                                | Status                                                                                                                                                                      |
+| ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Django version compatibility:        | [1.7](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/) ([cached docs](https://web.archive.org/web/20171025050612/https://docs.djangoproject.com/en/1.7/ref/)) |
```

Comment:
The 1.11 doc still up
https://docs.djangoproject.com/en/1.11/ref/templates/builtins/

Reply:
Absolutely; unfortunately, there are features in 1.11 that are not supported by pongo2, which undermines the stated aim to be compatible with django templates. Since the README link to django 1.7 is broken, this is a potential short term fix until the new version of pongo is released (another solution might be to post a note in the README explaining the situation and linking to the 1.11 docs, but without maintainer guidance, I did not want to overreach).

Reply:
@jprusik Can you please make a ticket with a list of features in 1.11 not supported here (that aren't specifically called out as intentionally not supported?)  Sometimes things get missed and that would at least make them available for review.